app/models/supervisor.rb

```diff
@@ -81,10 +81,13 @@ def recently_unassigned_volunteers
 #  receive_email_notifications   :boolean          default(TRUE)
 #  receive_reimbursement_email   :boolean          default(FALSE)
 #  receive_sms_notifications     :boolean          default(FALSE), not null
+#  refresh_token                 :string
```

Comment:
maybe make a new type of user, ApiUser?

Reply:
Created the [concern](https://github.com/rubyforgood/casa/blob/778758eb7081a6a883a623a74f18c7e70b1b0b1b/app/models/concerns/api.rb)